css/style.css

```diff
@@ -0,0 +1,57 @@
+body {
+  padding: 0;
+  margin: 0;
+  background-color: #ffe4e1;
+}
```

Comment:
- [ ] I strongly suggest you to stick to Figma template. Currently, there are some differences: 
## Figma design

![image](https://user-images.githubusercontent.com/77038610/143382914-f5385685-4ded-4596-869b-d4483c4c8017.png)


## Yours 

![image](https://user-images.githubusercontent.com/77038610/143382887-86ca7924-71b6-42cd-9434-0f6e49512e17.png)

- [ ] You can start from setting background color to `#fff` 🤗 
- [ ] [OTIONAL] : I suggest you to set `body{ width to 375px }`, this way the width will be same as on Figma, and you can make sure that your design is exactly same as the design on Figma 👍🏻

Reply:
done